modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/PodcastGridListFragment.kt

```diff
@@ -164,6 +181,7 @@ open class PodcastGridListFragment : BaseFragment(), Toolbar.OnMenuItemClickList
             linkView.visibility = View.VISIBLE
             linkTextView.text = linkTitle
             linkView.setOnClickListener {
+                analyticsTracker.track(AnalyticsEvent.DISCOVER_COLLECTION_LINK_TAPPED, mapOf(LIST_ID_KEY to (listUuid ?: NONE)))
```

Comment:
>  I like how you're using this "none" value here if the listUuid is missing. Curious why you decided to add this here.

`listUuid` passed to `PodcastGridListFragment` and `PodcastListFragment` can be null. It would have been best to pass `inferredId` property in this case. I made changes in `75fdef8` to get `inferredId` from the arguments passed to the fragment. 

I also made sure that I used `inferredId` at places [matching those with iOS](https://github.com/Automattic/pocket-casts-ios/search?q=inferredListId).

> In most other places we are instead just not sending the event at all if the listUuid is null (doing a text search for listUuid?.let seems to pull up a lot of examples. 

I ran through other places and found that `listUuid` is always present for them. The only exceptions I found (from admin area) are:
- `popular` and `trending` types lists which do not have `listUuid` and 
- `featured` type podcasts that have podcast uuid set on them. 

Moreover, `listUuid?.let` check was already present at most places for Firebase events. I didn't want to track additional events with `listUuid` set to `none`. Let me know if you feel otherwise. 🤯  

> I actually ran into an instance where an event wasn't getting fired because the listUuid was null. See https://github.com/Automattic/pocket-casts-android/pull/345#discussion_r986122711.

I answered it [here](https://github.com/Automattic/pocket-casts-android/pull/345#discussion_r986489681).